lib/src/flutter/flutter_test_configuration.dart

```diff
@@ -41,7 +40,7 @@ class FlutterTestConfiguration extends TestConfiguration {
     String targetAppPath = 'test_driver/app.dart',
   }) {
     return FlutterTestConfiguration()
-      ..features = [Glob(featurePath)]
+      ..features = [RegExp(featurePath)]
```

Comment:
When switching to `RegExp`, the `featurePath` value from `DEFAULT` configuration has to be changed:

```dart
String featurePath = 'test_driver\/features\/.*.feature',
```

Reply:
I think this would be a separate PR as it's not specifically related to null safety, right?

Reply:
Ah, no, nevermind you're right, since `RegExp` was introduced in gherkin 2.0